config/helm/chart/default/tests/Common/webhook/deployment-webhook_test.yaml

```diff
@@ -103,3 +120,111 @@ tests:
     asserts:
       - isNull:
           path: spec.template.spec.imagePullSecrets
+
+
+  - it: should exist on olm (but different)
+    set:
+      olm: true
+      operator.image: image-name
+    asserts:
+      - isKind:
+          of: Deployment
+      - equal:
+          path: metadata
+          value:
+            name: dynatrace-webhook
+            namespace: NAMESPACE
+            labels:
+              app.kubernetes.io/instance: RELEASE-NAME
+              app.kubernetes.io/name: RELEASE-NAME
+              dynatrace.com/operator: dynakube
+              helm.sh/chart: dynatrace-operator-0.4.0
```

Comment:
I removed the check for a specific version from the other test case, since it's causing trouble when updating the version. See #577